src/jobsautoreport/report.py

```diff
@@ -11,17 +11,20 @@
 logger = logging.getLogger(__name__)
 
 
-class JobStatesCount(BaseModel):
+class JobMetrics(BaseModel):
 
     successes: int
     failures: int
+    success_rate: float
     failure_rate: float
+    total_triggers: int
```

Comment:
same here:
```suggestion
    @property
    def total(self):
        return successes + failures
```
I wouldn't use "trigger" maybe "run" is a better name (trigger is an event to run something for me)

Reply:
or just "job" because it's what it is